src/libos/src/entry.rs

```diff
@@ -23,6 +24,7 @@ lazy_static! {
     static ref HAS_INIT: AtomicBool = AtomicBool::new(false);
     pub static ref ENTRY_POINTS: RwLock<Vec<PathBuf>> =
         RwLock::new(config::LIBOS_CONFIG.entry_points.clone());
+    pub static ref RESOLV_CONF_BYTES: RwLock<Vec<u8>> = RwLock::new(Vec::new());
```

Comment:
Using `String` seems better than `Vec<u8>`. First, the content of any well-formatted `resolve.conf` is a string, right? Second, `String` is much more convenient than `Vec<u8>` for debugging purposes.

Reply:
Done